typings/react-inspector/react-inspector.d.ts

```diff
@@ -0,0 +1,181 @@
+// Type definitions for react-inspector
+// Project: https://github.com/xyc/react-inspector
+// Definitions by: vidartf <https://github.com/vidartf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
We can also mention they come from https://github.com/vidartf/jupyterlab-kernelspy